extensions/ql-vscode/src/databases/local-databases-ui.ts

```diff
@@ -141,7 +141,9 @@ class DatabaseTreeDataProvider
     return item;
   }
 
-  public getChildren(element?: DatabaseItem): ProviderResult<DatabaseItem[]> {
+  public async getChildren(
```

Comment:
I had to make `getChildren` async (so that I could use `await this.databaseManager.setCurrentDatabaseItem`). It seems to be fine, but if there's any risk to doing this, let me know 🙈